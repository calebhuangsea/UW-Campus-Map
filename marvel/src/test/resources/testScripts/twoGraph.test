# test behavior when loading two graphs

LoadGraph onePath onePath.csv

FindPath onePath n1 n3

LoadGraph noParent noParent.csv

FindPath noParent n2 n1