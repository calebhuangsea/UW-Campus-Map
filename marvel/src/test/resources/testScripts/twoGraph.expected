# test behavior when loading two graphs

loaded graph onePath

path from n1 to n3:
n1 to n2 via edge1
n2 to n3 via edge2

loaded graph noParent

unknown: n2